components/ca-tel.js

```diff
@@ -0,0 +1,175 @@
+define([
+    './helpers/create-element.js',
+    './helpers/fire-event.js',
+    './ca-select.js',
```

Comment:
really not a fan of including one ca-component from another, I guess this might be one for refactoring?